<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LatticeDataDialogBase</class>
 <widget class="QDialog" name="LatticeDataDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lattice Data</string>
  </property>
  <widget class="QGroupBox" name="groupBox_wm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Weight Matrix</string>
   </property>
   <widget class="QWidget" name="widget_cw" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="rdButtonContWQ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Queen</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_cw</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rdButtonContWR">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rook</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_cw</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rdButtonContWB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bishop</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_cw</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_dw" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="txtThresh">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_knn" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="lblNN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Neighbors</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>10</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnOutput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output matrix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdButtonKNW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>K - Nearest Neighbors</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_wm</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rdButtonContW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Contiguity Weight</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_wm</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rdButtonDisW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance Weight</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_wm</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Moran I</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_mi">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>331</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Moran Index</string>
   </property>
   <widget class="QLineEdit" name="txtMoranI">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMoranIp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectField">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Field</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMoranIp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>p-value</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMoranIint">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpret</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMoranI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MoranÂ´s I</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBoxSelectField">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoranI">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scatter Plot</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtMoranIint">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="scatterplot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>scatterplot.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSelect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Polygon Layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>universidad.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdButton_nm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Moran I - New Matrix</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_calculate</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rdButton_sm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Moran I - Select Matrix</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_calculate</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="widget_sm" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblOutput_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select matrix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSelect">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelect">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="credits">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create by:</string>
   </property>
  </widget>
  <widget class="QLabel" name="credits_ac">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Andrea Castillo</string>
   </property>
  </widget>
  <widget class="QLabel" name="credits_jc">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Juan Carrillo</string>
   </property>
  </widget>
  <widget class="QLabel" name="credits_dr">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Diego Rodriguez</string>
   </property>
  </widget>
  <widget class="QLabel" name="credits_date">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2015</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_calculate"/>
  <buttongroup name="buttonGroup_wm"/>
  <buttongroup name="buttonGroup_cw"/>
 </buttongroups>
</ui>
